terraform/dns.tf

```diff
@@ -296,6 +296,11 @@ locals {
       type     = "CNAME"
       value    = "caliban.nixos.org"
     },
+    {
+      hostname = "tracker.security.nixos.org"
+      type     = "A"
+      value    = "188.245.41.195"
+    },
```

Comment:
Can we get IPv6 here as well?

Reply:
Possibly yes, I can take a look in a couple of hours.

Reply:
You can use this: https://github.com/NixOS/nixos-wiki-infra/blob/ac9dfe854f748bf8acedf394750d404aaa8dd075/targets/nixos-wiki.nixos.org/configuration.nix#L47

Reply:
Please make sure to use the ::1 suffixed address and not the network address.

Reply:
Done and I have confirmed the host is reachable via IPv6.